src/bundled.js

```diff
@@ -99,39 +99,46 @@ export function pluckQuery(... fields) {
   };
 }
 
-function containsField(obj, find) {
-    var args = find.split('.');
-
-  for (var i = 0; i < args.length; i++) {
-    if (!obj || !obj.hasOwnProperty(args[i])) {
-      return false;
+function findFieldOwner(obj, field) {
+  const nestedFields = field.split('.');
```

Comment:
If using `typeof obj[args[i]] === undefined`, we will not know whether the property is belong to the current `obj` or ancestor prototype (direct owner). To delete that property, we need to delete it from the direct owner. So, we need to use `hasOwnProperty`.

(About the const name `nestedFields`, I just make it more explicit but I can just use `args[i]` if you think it's more concise.)